labs/lab01/skel/win/3-bounds/Makefile.dynamic

```diff
@@ -1,23 +1,23 @@
 CC = cl
 L = link
-CFLAGS = /nologo /W3
+CFLAGS = /nologo /W3 /DDLL_IMPORTS
 
 all: bounds_dynamic.exe
 
+bounds_dynamic.exe: bounds.obj bounds_dynamic.lib
+	$(L) /nologo /out:$@ $**
 
-bounds_dynamic.exe: #TODO dependencies
-	#TODO create the executable file
+bounds_dynamic.lib: bounds_dynamic.dll
 
+bounds_dynamic.dll: min.obj max.obj
+	$(L) /nologo /dll /out:$@ /implib:bounds_dynamic.lib $**
 
-bounds_dynamic.dll: #TODO dependencies
-	#TODO create dynamic lib
-
-min.obj: #TODO 
-
-max.obj: #TODO 
-
-bounds.obj: #TODO 
-
+min.obj: min.c

```

Comment:
Dat fiind ca in laborator se mentioneaza regulile implicite [1], n-ar fi mai didactic daca le-am aplica si aici?

[1] https://ocw.cs.pub.ro/courses/so/laboratoare/laborator-01#folosirea_regulilor_implicite